frontend/src/components/shared/Modal/Modal.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2018-2020 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
I think we can ax this file? 50% of this looks like it's covered in our body default, BaseWeb's theme and/or JS. Given that we're doing styling in JS also, can we standardize one way or the other? Fine to leave as is if this is a transitional state.

Reply:
Yes. That's easy to address.